app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -1,7 +1,7 @@
-module Idv
-  class GpoController < ApplicationController
+module Idv::ByMail
```

Comment:
can we break this out into a nested namespace?
```suggestion
module Idv
  module ByMail
```